pkg/pipeline/run.go

```diff
@@ -37,13 +37,13 @@ func Run(pc *config.Provider, rootDir string) { // nolint:gocyclo
 	// Group resources based on their Group and API Versions.
 	resourcesGroups := map[string]map[string]map[string]*config.Resource{}
```

Comment:
A more explicit comment would help here, maybe using a syntax like:
`<API group prefix/short group>` -> <API version> -> <resource name>` and maybe with a specific example for a resource.